### PR TITLE
[18.0][MIG] sale_commercial_partner: Migration to 18.0

### PR DESCRIPTION
**Pull request by LucasTran380381**

### CODE REVIEW

**Comment by thienvh332** [comment]
Hi @LucasTran380381 ,
- The first time we run pre-commit we will not fix anything this time (Even if it has errors) and commit with message "IMP] sale_commercial_partner: pre-commit auto fixes"
- The second commit will be the migration commit. Here we will change the code accordingly.

Your edit in the pre-commit commit leads to unwanted compare changes in the second commit. This results in a bad history. Can you fix it?

File: sale_commercial_partner/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    commercial_partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Customer Entity",
+        related="partner_id.commercial_partner_id",
+        store=True,
+        index=True,
```
Comment (thienvh332) [inline sale_commercial_partner/models/sale_order.py]: See `https://github.com/odoo/odoo/pull/127353`
```suggestion

```

Comment (LucasTran380381) [reply sale_commercial_partner/models/sale_order.py]: I fixed

File: sale_commercial_partner/checklog-odoo.cfg
Comment (thienvh332) [inline sale_commercial_partner/checklog-odoo.cfg:1]: Where is the content in this file?

Comment (LucasTran380381) [reply sale_commercial_partner/checklog-odoo.cfg:1]: move this file to root repo

File: checklog-odoo.cfg
Comment (thienvh332) [inline checklog-odoo.cfg:1]: I mean: it's empty. What do you need an empty file for? :laughing:

Comment (thienvh332) [reply checklog-odoo.cfg:1]: Hehe, I review too fast :laughing:

**Review by thienvh332 — APPROVED** [review]

**Pull request closed without merging**